src/frontend/src/lib/components/ui/DropdownButton.svelte

```diff
@@ -0,0 +1,28 @@
+<script lang="ts">
+	import { IconExpandMore } from '@dfinity/gix-components';
+
+	export let disabled = false;
+	export let button: HTMLButtonElement | undefined = undefined;
+	export let ariaLabel: string;
+	export let opened = false;
+</script>
+
+<button
+	class="text-secondary min-w-72 justify-between gap-2 rounded-xl border border-light-grey bg-white px-4 py-3 text-left font-medium leading-5 text-inherit hover:border-blue-ribbon"
+	bind:this={button}
+	on:click
+	aria-label={ariaLabel}
+	{disabled}
+	class:opacity-50={disabled}
+	class:border={opened}
+	class:border-blue-ribbon={opened}
+>
+	<slot />
+	<div
+		class="transform transition-transform duration-300 ease-in-out"
+		class:scale-y-[-1]={opened}
```

Comment:
Out of curiosity, what does `scale-y-` do? I don't think we have it yet and I don't know it 😅.

Reply:
this is the one that make the arrow icon "flip": instead of rotate, I think it was better to make it flip when the dropdown is open, WDYT?

Reply:
Ah now I get it. Looks cool 👍

Reply:
but you are right, it is not needed when not opened, and it can be used `-scale-y-100` that is the built-in